pkg/extensions/sync/sync.go

```diff
@@ -212,11 +214,11 @@ func syncRegistry(ctx context.Context, regCfg syncconf.RegistryConfig,
 		return err
 	}
 
-	log.Info().Msgf("filtering %d repos based on sync prefixes", len(catalog.Repositories))
+	log.Info().Int("reposNumber", len(catalog.Repositories)).Msg("filtering repos based on sync prefixes")
```

Comment:
New message doesn't make much sense. Let's keep the old one.

OR

```suggestion
	log.Info().Int("repo count", len(catalog.Repositories)).Msg("filtering repos based on sync prefixes")
```